controllers/storagerequest/storagerequest_controller.go

```diff
@@ -48,6 +48,7 @@ import (
 const (
 	storageRequestFinalizer    = "ocs.openshift.io.storagerequest"
 	forceDeletionAnnotationKey = "rook.io/force-deletion"
+	blockPoolNameLabel         = "blockPoolName"
```

Comment:
If we are having this here, could we have something like `ocs.openshift.io/cephblockpool-name`